### PR TITLE
CVE-2016-6186

### PR DESCRIPTION
**Pull request by rxv3774**
Completed research about vulnerability.

### CODE REVIEW

File: cves/CVE-2016-6186.yml [hunk 1]
```diff
@@ -202,8 +219,8 @@ ipc:
     software system reads is another form of IPC.
 
     Answer should be boolean.
-  answer:
-  note:
+  answer: false
+  note: None
```
Comment (zxw4320) [inline cves/CVE-2016-6186.yml]: May want more reasoning rather than just 'None'.

File: cves/CVE-2016-6186.yml [hunk 2]
```diff
@@ -169,8 +181,10 @@ subsystem:
     In the name field, a subsystem name (or an array of names)
 
     e.g. clipboard, model, view, controller, mod_dav, ui, authentication
-  answer:
-  name:
+  answer: |
+    The vulnerability was in the ui component because of the fact that this input field
+    that requires sanitization may be found in the frontend component.
+  name: User Interface
```
Comment (zxw4320) [inline cves/CVE-2016-6186.yml]: UI is fairly generic. Is this part of a specific subsystem within the UI?

File: cves/CVE-2016-6186.yml [hunk 3]
```diff
@@ -49,37 +49,42 @@ description_instructions: |
...
-  note: This VCC was discovered automatically via archeogit.
-- commit: 7b21bfc0745f92f83a6826a3d3e58797fb74e326
-  note: This VCC was discovered automatically via archeogit.
-- commit: c80d85d4ad05ee905321e410e1a3e7a49d730233
-  note: This VCC was discovered automatically via archeogit.
-- commit: f5cfd09c257e5a4ed6d01d32505c48b02e3860d5
-  note: This VCC was discovered automatically via archeogit.
+- commit: 07988744b347302925bc6cc66511e34224db55ab
+  note: This VCC introduced the vulnerability stated above.
 upvotes_instructions: |
```
Comment (zxw4320) [inline cves/CVE-2016-6186.yml:89]: upvotes: 7

File: cves/CVE-2016-6186.yml [hunk 4]
```diff
@@ -49,37 +49,42 @@ description_instructions: |
...
-description:
+description: |
+  A pop-up related function was made vulnerable to cross site scripting
+  attacks due to the fact that the inputs that were being supplied by the user were
+  not being sanitized properly. This allowed attackers to enter dangerous scripts
+  which could steal authentication credentials from the user, and even change how
+  the site is being rendered to the user.
+
+  The fix to this issue was referncing the content in the text instead of the content
+  within the HTML, this prevents the execution of the data.
```
Comment (zxw4320) [inline cves/CVE-2016-6186.yml:61]: Due to user supplied inputs not being sanitized properly.
Fix for this rather than to this.

Slight simplifications.

Comment (rxv3774) [reply cves/CVE-2016-6186.yml:61]: Simplified it a bit

File: cves/CVE-2016-6186.yml (hunk 3 above)
Comment (rxv3774) [reply cves/CVE-2016-6186.yml:89]: updated

File: cves/CVE-2016-6186.yml (hunk 2 above)
Comment (rxv3774) [reply cves/CVE-2016-6186.yml]: Specified the control inputs component of the UI

File: cves/CVE-2016-6186.yml (hunk 1 above)
Comment (rxv3774) [reply cves/CVE-2016-6186.yml]: Updated

**Pull request merged**